mllib/src/main/scala/org/apache/spark/ml/tree/impl/GradientBoostedTrees.scala

```diff
@@ -309,6 +309,7 @@ private[spark] object GradientBoostedTrees extends Logging {
 
     var m = 1
     var doneLearning = false
+    var pre_predError=predError
```

Comment:
Thank you for reminding me  of these  basic rules. I will read the contributing guide and  modify the code style.